script/client/conf/registry.conf

```diff
@@ -95,6 +95,7 @@ config {
     apolloMeta = "http://192.168.1.204:8801"
     namespace = "application"
     apolloAccesskeySecret = ""
+    cluster = "seata"
```

Comment:
This did not previously have a default value and it is suggested that it be changed to `""`.

Reply:
done